web-client/src/presenter/actions/broadcastLogoutAction.ts

```diff
@@ -6,11 +8,14 @@
  */
 export const broadcastLogoutAction = async ({
   applicationContext,
+  get,
   props,
 }: ActionProps) => {
   // for some reason this causes jest integration tests to never finish, so don't run in CI
-  if (!process.env.CI && !props.skipBroadcast) {
+  if (!props.skipBroadcast) {
```

Comment:
See my above comment: https://github.com/ustaxcourt/ef-cms/pull/5176/files#r1695792563. I highly suspect that this was failing because of our !process.env.CI check in app.tsx.

Reply:
Aaaand I forgot to move the "for some reason" comment lol. Will do on next push.